GA/multi_point_problem_definition.py

```
@@ -433,11 +443,61 @@ def ComputeMTFLOInputs(self,
+            # Set parameters equal to the config values in case of a crash so that the constraint/objective value calculations do not crash
+            self.Lref = config.BLADE_DIAMETERS[0]
+            self.duct_variables = copy.copy(config.DUCT_VALUES)
+            self.centerbody_variables = copy.copy(config.CENTERBODY_VALUES)
+            self.blade_blading_parameters = copy.copy(config.STAGE_BLADING_PARAMETERS)
+            self.blade_design_parameters = copy.copy(config.STAGE_DESIGN_VARIABLES)
+
+        return input_generated
+    
+
+    def _check_output_validity(self,
+                               outputs: dict[str, Any]) -> bool:
```

Ruff (0.11.9)

Undefined name `Any`

(F821)